Zend/zend_vm_def.h

```diff
@@ -1005,7 +1005,7 @@ ZEND_VM_COLD_HELPER(zend_undefined_function_helper, ANY, ANY)
 	HANDLE_EXCEPTION();
 }
 
-ZEND_VM_HANDLER(28, ZEND_ASSIGN_OBJ_OP, VAR|UNUSED|THIS|CV, CONST|TMPVAR|CV, OP)
+ZEND_VM_HANDLER(28, ZEND_ASSIGN_OBJ_OP, VAR|UNUSED|THIS|CV, CONST|TMPVAR|CV, OP, SPEC(DELAYED))
```

Comment:
``SPEC(DELAYED)`` is going to double the number of related handlers. Right?
I see ~40MB zend_execute.o size increase.

Reply:
It shouldn't. `SPEC(DELAYED)` should add a single, otherwise unspecialized handler. It's possible that this behavior is still buggy, but that's the idea.